src/infrastructure/api/fetch_Items.ts

```diff
@@ -1,12 +1,12 @@
 import axios from 'axios';
 
-const apiCallFunc = async() => {
+const fetchItems = async() => {
```

Comment:
much better naming. Best is actually `listItems`

Reply:
if you are getting  a list of seomthing, please use `list`. If you are getting only one thing, use `retrieve`

So for example: `listItems` and `retrieveItem`. This is how we will name things in our codebase